THANKS.txt

```diff
@@ -130,7 +130,9 @@ Jörg Dietrich for the k-sample Anderson Darling test.
 Blake Griffith for improvements to scipy.sparse.
 Andrew Nelson for scipy.optimize.differential_evolution.
 Brian Newsom for work on ctypes multivariate integration.
-Nathan Woods for advising on multivariate integration and building unit tests.
+Nathan Woods for work on multivariate integration. His work from 31 Aug 2015
+    to the present is subject to the disclaimer at:
+    https://github.com/losalamos/Gazebo/blob/master/LICENSE.
```

Comment:
I agree wholeheartedly. The lawyers do not, however. I intend to keep pushing them to come up with something better (something in the commit message, maybe?), but this is already better than what they started with. 
I can include some basic description in the THANKS.txt, in case the link breaks, if that would be helpful. 



    _____________________________
From: Ralf Gommers <notifications@github.com>
Sent: Saturday, February 27, 2016 4:54 PM
Subject: Re: [scipy] Enhancements and cleanup in scipy.integrate (#5892)
To: scipy/scipy <scipy@noreply.github.com>
Cc: Nathan Woods <charlesnwoods@gmail.com>


    

In THANKS.txt:   > @@ -130,7 +130,9 @@ Jörg Dietrich for the k-sample Anderson Darling test.>  Blake Griffith for improvements to scipy.sparse.>  Andrew Nelson for scipy.optimize.differential_evolution.>  Brian Newsom for work on ctypes multivariate integration.> -Nathan Woods for advising on multivariate integration and building unit tests.> +Nathan Woods for work on multivariate integration. His work from 31 Aug 2015> +    to the present is subject to the disclaimer at:> +    https://github.com/losalamos/Gazebo/blob/master/LICENSE.   

Hmm, such a link doesn't seem very future-proof. And it's basically the same license as SciPy uses (3-clause BSD) except for the sentence where it mentions the contract number under which it is produced. It would be simpler to just say that your contributions are copyright LANL.   

—
Reply to this email directly or view it on GitHub.